_posts/blog/2018-09-12/2018-09-12-how-to-get-more-buyers-on-openbazaar-with-categories-and-tags.markdown

```diff
@@ -0,0 +1,145 @@
+---
+title: "How To Get <span class='post-title-extra'>More Buyers</span> on OpenBazaar with <span class='post-title-extra'>Categories and Tags</span>"
+layout: post
+date: '2018-09-12 00:30:00 -0300'
+social_title: 'How To Get More Buyers on OpenBazaar with Categories and Tags'
+social_description: 'A little extra work on your OpenBazaar store can give buyers an exceptional experience to make them want to visit again and again! Here is how to use tags and categories to help users find and browse your store with ease.'
+social_image: OpenBazaar demo store - How To Get More Buyers on OpenBazar with Categories and Tags.png
+---
+
+_by Jason Hotelling, OpenBazaar Front End Developer_
+
+Hello! We hope you're having a great time selling and sharing on the OpenBazaar platform. 
+
+OpenBazaar stores boast the fastest setup time of any e-commerce platform. With very little effort, you can set up your store and add listings in minutes. Then, a little extra work can give buyers an exceptional experience to make them want to visit again and again! 
+
+Today we are going to talk about how to use tags and categories to help users find and browse your store with ease.
+
+### Before We Get Started
+
+If you haven't already, please follow our guides to get started with OpenBazaar. They contain instructions and tutorials that span from _Download_ through _Browsing_ and _Selling_.
+
+[The Beginner's Guide to Buying Goods, Services and Cryptocurrency on Openbazaar](https://openbazaar.org/blog/The-Beginners-Guide-to-Buying-Goods-Services-and-Cryptocurrency-on-OpenBazaar/)
+
+If you are new, we also recommend watching our [OpenBazaar 2.0 Tutorial](https://www.youtube.com/watch?v=VzlerMJMOu8) on YouTube.
+
+{% include modules/embeded-video.html url="https://www.youtube.com/embed/VzlerMJMOu8"%}
+
+### Step 1: Tags Are for Search
+
+**Tags** are what the search engines use to determine which listings to show to a potential buyer so they are very important. Most users will discover your store through a listing that they find in search. For example, if a user is looking for t-shirts, they first navigate to the **Discover** tab in OpenBazaar. Then they will select a search provider and enter a search term such as "t-shirt" or "t shirt". In order for potential buyers to find _your shirts_, you must enter any relevant tags on your listings. 
```

Comment:
I would say "Tags are one of the things search engines use..." since they are not the only thing.

Search works based on the title, description, slug, categories, and condition (for example, searching for "new hat" matches titles of "hat" with the condition "new").

Reply:
Same comment on the last sentence, we don't want to imply the only way for a user to find your listing is if it has a tag that exactly matches what they searched for. The logical response to that is to add infinite tags to every listing to cover every possible search term.

Search works just fine with listings that have no tags.

Reply:
I was not aware of this, I had the impression search used ONLY tags and Jason didn't push back here either...thank you for clarifying!